pydis_site/apps/home/views/home.py

```diff
@@ -29,6 +29,12 @@ class HomeView(View):
         "python-discord/django-simple-bulma",
     ]
 
+    def __init__(self):
+        """Clean up stale RepositoryMetadata."""
+        for cached_repo in RepositoryMetadata.objects.all():
+            if cached_repo.repo_name not in self.repos:
+                cached_repo.delete()
```

Comment:
I think it would be much more efficient to filter the repositories in SQL, using Django's ORM:
```suggestion
        for cached_repo in RepositoryMetadata.objects.exclude(repo_name__in=self.repos):
            cached_repo.delete()
```

Reply:
Sure, I don't mind this. The efficiency is pointless here because there's only ever like 6 rows in this table, but I think I like your code better and it's fewer lines.